### ISSUE TITLE
[Suggestion] Clean up source code directory tree

### ISSUE THREAD

**Issue by DanielEckl** [rafael2k/darkice#197]
Hello Rafael and all, thank you for still being here, investing time and experience into this great little piece of software. I appreciate this a lot.

Maybe it would be time to clean up the code by splitting up darkice, darkice-gui and darkice-website into their own repositories.

Additionally I'd like to suggest to get rid of the folder-style tags, branches and trunk which may or may not be a relic from an ancient code versioning system. I checked the files in darkice/trunk and the history goes back 24 years, so I assume this is all we need. As a result the release tarballs automatically will look clean. I hope this is as simple as I assume.

**Comment by marillat** [comment]
I agree with @DanielEckl  the current 1.5 release tarball is unusable.

**Comment by rafael2k** [comment]
Could you please help me with this? I can add permissions to the repo.

**Comment by DanielEckl** [comment]
Not at the moment, but I will tell here when I think I can do.

**Comment by marillat** [comment]
As I'm not too familiar with hithub I'll let @DanielEckl  do the work :)
